vg/geom.go

```diff
@@ -0,0 +1,58 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package vg
+
+// A Point is a location in 2d space.
+//
+// Points are used for drawing, not for data.  For
+// data, see the XYer interface.
+type Point struct {
+	X, Y Length
+}
+
+// Dot returns the dot product of two points.
+func (p Point) Dot(q Point) Length {
+	return p.X*q.X + p.Y*q.Y
+}
+
+// Plus returns the component-wise sum of two points.
+func (p Point) Plus(q Point) Point {
```

Comment:
Since we're exposing these for the first time, how about we choose more commonly used names.

`Add`